portafly/README.md

```diff
@@ -1,6 +1,31 @@
 # PortaFly
 A React SPA crafted with the ambitious purpose of replacing Porta UI, currently residing inside the Rails App.
 
+## Getting up and running
+1. Clone the `porta` repository to your local machine
+2. Create a `.env` file:
+  * Create a new file in the `/portafly` root folder, and name it `.env`
+  * Paste `REACT_APP_API_HOST="https://multitenant-admin.preview01.3scale.net"` into it, and Save
```

Comment:
This is very unstable. We use Preview for testing and many times this API host won't be available. I think that it is important to point this out.

Reply:
Btw, it should be possible to use the local porta master/tenant account as API host as well. In fact, that will be more stable.

Reply:
we shouldn't make our preview env public